lib/doorkeeper/openid_connect/config.rb

```diff
@@ -91,8 +91,9 @@ def extended(base)
 
       extend Option
 
-      option :jws_private_key
+      option :encryption_method
       option :issuer
+      option :jws_private_key
```

Comment:
Should we rename this to `signing_key` and do the deprecation warning for `jws_private_key` [as discussed here](https://github.com/doorkeeper-gem/doorkeeper-openid_connect/issues/25#issuecomment-267634632)?